### DISCUSSION TITLE
Unable to create folder for config autosave - error

### DISCUSSION THREAD

**Discussion by DenuxPlays** [nextcloud/all-in-one#1810] [Questions] [answered]
Hey,
I am getting this error in my nextcloud mastercontainer logs:
![image](https://user-images.githubusercontent.com/83671398/213515654-bfee3b82-538a-4807-997e-4103e989f59f.png)
I do not know why this error is appearing.

**Comment by szaimen** [comment, marked as answer]
Hi, do you have a specific problem or why are you reporting this?

**Reply by DenuxPlays** [reply]
> Hi, do you have a specific problem or why are you reporting this?

Sometimes when i log into my nextcloud I just get an html error 500 and this was the only thing I could find in the logs.
Maybe it has something to do with it.

**Reply by szaimen** [reply]
No, I don't think so. Also this is a log entry from the mastercontainer and not from the apache container or the nextcloud container so definitely unrelated to Nextcloud login.

**Reply by DenuxPlays** [reply]
> No, I don't think so. Also this is a log entry from the mastercontainer and not from the apache container or the nextcloud container so definitely unrelated to Nextcloud login.

Yeah already thought it...
Ok but thank you then I am going back to debuggin this. :)

**Reply by DenuxPlays** [reply]
Found the problem. I set a second account (the acc I tried to log into) as an admin and when I removed the group everything works fine.
